src/mappings/CciNistMappingData.ts

```diff
@@ -0,0 +1,5100 @@
+export const data = {
+  'CCI-000001': 'AC-1 a 1',
```

Comment:
Where does this mapping come from? (Specifically, the CCIs) I heard someone conversation about these tags in a Vulcan talk today and whether or not you can get the `AC-1` type of tags with what is automatically populated from an SRG. I think I need to know more.

Reply:
These are taken from DISA's website [here](https://public.cyber.mil/stigs/cci/) and is converted with [mitre/heimdall2@libs/hdf-converters/data/converters/xml2json.ts](https://github.com/mitre/heimdall2/blob/master/libs/hdf-converters/data/converters/xml2json.ts)